src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -304,9 +304,13 @@ impl Action for IngestAction {
 
         let mut futs = FuturesUnordered::new();
 
-        for _n in 0..self.repeat {
+        for iteration in 0..self.repeat {
             for row in &self.rows {
-                let mut row = row.as_bytes();
+                let row_replaced = row.replace(
+                    &INGEST_ITERATION_RUNTIME_VAR_DECORATED,
+                    &iteration.to_string(),
+                );
+                let mut row = row_replaced.as_bytes();
```

Comment:
Folding those two lines into one invites "temporary value is freed at the end of this statement"